tests/impl.h

```diff
@@ -7,531 +7,533 @@
     case it_##c:                  \
         ret = test_##c(*this, i); \
         break;
-#define INTRIN_FOREACH(TYPE)       \
```

Comment:
Why did you change this a lot?

Reply:
Because the new intrinsics `_mm_[get|set]_denormals_zero_mode` are added, and they are longer than other intrinsic names.
The changing is done by applying the coding style checking command.